src/Nancy.Tests/xUnitExtensions/RecordAsync.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Nancy.Tests.xUnitExtensions
+{
+    using System;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public static class RecordAsync

```

Comment:
Be nice to not have to repeat this, but probably the lesser of two evils compared to adding references.